scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -216,9 +216,9 @@ os_dependencies() {
       if [[ "${DISTRO}" =~ Rocky ]]; then
         #RockyLinux 8/9
         pkg_list="${pkg_list} --enablerepo=devel,crb libusbx ncurses-compat-libs pkgconf-pkg-config"
-      elif [[ "${DISTRO}" =~ "Red Hat" ]] || [[ "${VERSION_ID}" =~ "8" ]]; then
+      elif [[ "${DISTRO}" =~ "Red Hat" ]] && [[ "${VERSION_ID}" =~ "8" ]]; then
```

Comment:
As mentioned in comment on linked ) PR, minor thing - but can we 219 and 221 - and simply substitute VERSION_ID (instead of specifying 8 vs 9) in the package install command